tests/gnrc_dhcpv6_client/tests-with-config/check-config.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/sh
+#
+# check_config.sh
+# Copyright (C) 2021 Martine Lenders <mail@martine-lenders.eu>
+#
+# Distributed under terms of the MIT license.
+#
+
+ip link show dev "${IFACE}" > /dev/null
+RESULT=$?
+
+if [ $RESULT -ne 0 ]; then
+    echo "You can create \"${IFACE}\" by using e.g." \
```

Comment:
```suggestion
    echo "You may be able to create \"${IFACE}\" by using e.g." \
```
sometimes the test might be used with non tap devices